src/openforms/logging/logevent.py

```diff
@@ -471,6 +470,8 @@ def submission_logic_evaluated(
     initial_data: dict,
     resolved_data: dict,
 ):
+    from openforms.submissions.logic.rules import get_targeted_components
```

Comment:
could we maybe move that into a dedicated `openforms.submissions.logic.log_utils` module? I'd like to keep the other modules focused on effective execution.